src/app/recent/nearby/page.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import MapComponent from "../../components/map";
+import { getNearbySightings } from "@/app/actions";
+
+export type NearbySightingsResponse = Array<{
+  speciesCode: string;
+  comName: string;
+  sciName: string;
+  locId: string;
+  locName: string;
+  obsDt: string;
+  howMany: number;
+  lat: number;
+  lng: number;
+  obsValid: boolean;
+  obsReviewed: boolean;
+  locationPrivate: boolean;
+}>;
+
+export default function NearbySightingsPage() {
+  const [nearbySightings, setNearbySightings] =
+    useState<NearbySightingsResponse>();
+
+  const handleDragEnd = (longitude: number, latitude: number) => {
+    getNearbySightings(longitude, latitude).then((response) => {
+      setNearbySightings(response);
+    });
+  };
+
+  useEffect(() => {
+    getNearbySightings(-1.688074, 53.185335).then((response) => {
```

Comment:
This hard coded value means the map component will always reload centred just outside of Bakewell.

Reply:
It doesn't! But, I'm thinking it potentially could be set based off the users last centre LatLong. We could potentially store cookies 😈 or have signed in users.

Reply:
Sure,

What's that got to do with fetching the nearby sightings from bakewell? 

The map's starting point isn't controlled by this function call, it's controlled independently using its own starting center configuration. So is it more accurate to say that this hard coded value fetches sightings from the same hard-coded starting point of the map so that when you land on the page, you see markers in the map? 

Then, when we change the starting point for the map we'll have to remember to pass it both here _and_ into the Map component in some way so that they're in-sync.

Reply:
I realise now I thought we were talking about the code in the map component when we obviously are not.

Your big paragraph does summarise it well, and yes that makes total sense. We need to remember to change the hard-coded values here and in the map component's centre values!